Classes/PBCommitList.m

```diff
@@ -43,7 +43,10 @@ - (void)keyDown:(NSEvent *)event
 	}
 	else if ([character rangeOfCharacterFromSet:[NSCharacterSet characterSetWithCharactersInString:@"jkcv"]].location == 0)
 		[webController sendKey: character];
-	else
+	else if (([character characterAtIndex:0] == NSDownArrowFunctionKey)
+			 && [event modifierFlags] & NSControlKeyMask) {
+		[controller selectParentCommit:self];
+	} else
```

Comment:
@tiennou : I think I see your point, but I’ll still bring the usual GUI argument that an important point of a GUI is to make commands discoverable. And I wouldn’t know how to discover this command which sounds like it may come handy.